src/OSPSuite.Engine/Serialization/SimModelExporter.cs

```diff
@@ -49,7 +49,8 @@ private MatlabCodeWriteMode writeModeFrom(MatlabFormulaExportMode formulaExportM
 
       private SimulationExport createSimulationExport(IModelCoreSimulation simulation, SimModelExportMode simModelExportMode)
       {
-         var simulationExport = _createExportModelVisitor.CreateExportFor(simulation.Model, simModelExportMode);
+         var simulationExportCreator = _simulationExportCreatorFactory.Create();
```

Comment:
Here new instance created everytime instead of having a reference to a single export